lib/api/manipulation.js

```diff
@@ -410,23 +410,27 @@ exports.wrapAll = function (wrapper) {
       wrapper = wrapper.call(this[0]);
     }
 
-    var wrap = this._make(wrapper);
+    // trim whitespace
+    if (typeof wrapper === 'string') {
+      wrapper = wrapper.trim();
```

Comment:
Can you add a test that breaks without this? I'm not sure if it is necessary 🤔

Reply:
If html string is used, and first characters are any white space character - parser will generate, text node.

It may be overkill, but I managed encounter this during my first tests. I like to test it further, but it should avoid some errors.